sdk/dapp-kit/src/utils/walletUtils.ts

```diff
@@ -5,19 +5,20 @@ import type {
 	MinimallyRequiredFeatures,
 	Wallet,
 	WalletWithFeatures,
+	WalletWithRequiredFeatures,
 } from '@mysten/wallet-standard';
 import { getWallets, isWalletWithRequiredFeatureSet } from '@mysten/wallet-standard';
 
 export function getRegisteredWallets<AdditionalFeatures extends Wallet['features']>(
 	preferredWallets: string[],
-	requiredFeatures?: (keyof AdditionalFeatures)[],
+	walletFilter?: (wallet: WalletWithRequiredFeatures) => boolean,
```

Comment:
@Jordan-Mysten since you are mostly looking at vercel preview for this PR, just want to flag these changes are in this PR too